layout/js/map.js

```diff
@@ -98,23 +98,40 @@ var animate_bars = function(data) {
   $.each(data, function(prop, val) {
     var myYear = prop;
     var color = colors[category];
+
     var scale = val[category][0]["barScale"];
     var value = val[category][0]["value"];
-    var style = {
-      "background": color,
-      "transform": "scale3d(1," + scale + ",1)",
-      "transform-origin": "100% 100%",
-      "transition": "all " + transitionTime + " ease-in-out"
-    };
+
     var bar = $("#bar-" + myYear);
+    var style = undefined;
+
     if (bar !== undefined) {
+      if (isNaN(scale)){ 
+        style = {
```

Comment:
I guess we could have just said `isNaN(scale) then scale = 0` and this would work. Didn't see that the first time.